docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -2855,7 +2868,55 @@ Netlogin (802.1X)
   configure netlogin ports 1-48 mode port-based-vlans
   configure netlogin ports 1-48 no-restart
 
-NOTE: You can mix the MAC Authentication and 802.1X on the same switchport. If the device fails 802.1X authentication, it will roll back to the MAC Authentication.
+NOTE: You can mix the MAC Authentication and 802.1X on the same switchport. If the device fails 802.1X authentication, it will fallback to the MAC Authentication.
+
+Policy based access
++++++++++++++++++++
+
+You can assign policies defined on the switch via PacketFence.
+
+First define your policy in the switch:
+
+  configure policy profile 1 name "gaming" pvid-status "enable" pvid 3521 untagged-vlans 3521
+  configure policy profile 2 name "guest" pvid-status "enable" pvid 3522 untagged-vlans 3522
+  configure policy maptable response both
+  configure policy vlanauthorization enable
+
+Next, in PacketFence, enable 'Role by Switch Role' your the switch configuration and assign the policies to the roles there. They will be returned inside the Filter-Id attribute.
```

Comment:
Next, in PacketFence, enable 'Role by Switch Role' in your the switch configuration and assign the policies to the roles there. They will be returned inside the Filter-Id attribute.